gatsby/content/projects/clients/quadrix.mdx

```diff
@@ -0,0 +1,66 @@
+---
+layout: projectimage
+title: quadrix.chat
+slug: quadrix
+categories:
+ - client
+description: Minimal Matrix client available in all main app stores
+author: Jean-François Alarie
+home: https://quadrix.chat
+language: Typescript
+license: GPL-3.0-or-later
+repo: https://github.com/alariej/quadrix
+room: "#quadrix:matrix.org"
+thumbnail: /docs/projects/images/quadrix-screenshot.png
+screenshot: /docs/projects/images/quadrix-screenshot.png
+maturity: Released
+platforms:
+    - Linux
+    - Mac
+    - Windows
+    - Web
+    - Android
+    - iOS
+    - Mobian
+featured: true
+features:
+    Room directory: yes
+    Room tag showing: no
+    Room tag editing: no
+    Search joined rooms: no
+    Room user list: yes
+    Display Room Description: yes
+    Edit Room Description: no
+    Highlights: no
+    Pushrules: no
+    Send read markers: yes
+    Display read markers: yes
+    Sending Invites: yes
+    Accepting Invites: yes
+    Typing Notification: no
+    E2E: no
+    Replies: yes
+    Attachment uploading: yes
+    Attachment downloading: yes
+    Send stickers: no
+    Send formatted messages (markdown): no
+    Rich Text Editor for formatted messages: no
+    Display formatted messages: no
+    Redacting: yes
+    Multiple Matrix Accounts: no
+    New user registration: yes
+    voip: no
+    Reactions: no
+    Message editing: no
+    Room upgrades: no
+    Localisations: 3
+    SSO Support: no
+---  
+The current focus is on creating a simple and stable chat application
+which runs on all platforms. The Quadrix apps are available on Linux
+(snap and flatpak packages for amd64, arm64, armhf), Windows, Mac,
+Android and iOS. It has been tested on a PinePhone running Mobian/Phosh.
```

Comment:
Cool to see PinePhone getting some more support! 🙂

Reply:
The flatpak runs surprisingly well on PinePhone. Main problem is the phone's camera, which refuses to start when launching a Jitsi call :-(